terraform/groups/ecs-service/locals.tf

```diff
@@ -10,9 +10,9 @@ locals {
   eric_port                       = "10000"
   docker_repo                     = "company-profile-api"
   kms_alias                       = "alias/${var.aws_profile}/environment-services-kms"
-  lb_listener_rule_priority_links = 40
+  lb_listener_rule_priority_links = 9
```

Comment:
as discussed in slack, this needs to remain a lower number than the main officers/appointments endpoint (previously 40 when that was 41 - now thats changing to 4 this should change to 3)

Reply:
Updated in [a8b4abc](https://github.com/companieshouse/company-profile-api/pull/236/commits/a8b4abc1858ba917b8adf63e06a1861dd8d827e9)